cypress/e2e/pricing.cy.ts

```diff
@@ -0,0 +1,128 @@
+/// <reference types="cypress" />
+
+import { STRIPE_PRODUCT } from '../../pages/api/stripe/stripeProducts';
+
+//Check if the buttons works properly
+describe('Pricing landing', () => {
+  const DRIVE_WEB_URL = Cypress.env('DRIVE_WEB_URL');
+  const TWOTB_OFF_COUPON = 'P8PSpVs6';
+
+  const url = ({ planId, couponCode, mode }: { planId: string; couponCode?: string; mode?: string }) => {
+    return `${DRIVE_WEB_URL}/new?redirectUrl=${encodeURIComponent(
+      DRIVE_WEB_URL + '/checkout-plan',
+    )}${encodeURIComponent(
+      `?planId=${planId}${couponCode ? '&couponCode=' + couponCode : ''}&mode=${mode ? mode : 'subscription'}`,
+    )}`;
+  };
+
+  describe('Free plan button', () => {
```

Comment:
See [the AAA pattern that we advise in our internal docs](https://inxt.atlassian.net/wiki/spaces/T/pages/25460784/How+to+Testing+Guideline#Categorize-tests-under-at-least-2-levels) that could make the explanations of the test more understandable. As an example:

`describe('Pricing page', () => {
    describe('When free plan button is clicked', () => {
       it('Then there is a redirection to ...', () => {
            // ...
       });
    });
});`

Then, when the pipeline fails, the test that is failing clearly states what is going on. Reading the current test, some questions arise:

- Free plan button, 10GB inside, is there another free plan button apart from it? 
- Why is the 10GB not 10GB anymore?